LICENSE

```diff
@@ -531,1025 +526,172 @@ That's all there is to it!
 
 ==========================================================================
 
-                    GNU GENERAL PUBLIC LICENSE
+
+ GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
Should be centered to match https://www.gnu.org/licenses/lgpl-3.0.en.html#license-text

Reply:
Done :)